packages/core/src/components/blind/blind.scss

```diff
@@ -21,6 +21,10 @@
   border-radius: var(--theme-blind--border-radius);
   overflow: hidden;
 
+  :focus {
```

Comment:
In general we try to use focus-visible instead of focus. Would this lead to issues with regards to a11y?
Also we have some SASS mixins defined for that:
@import 'mixins/hover';
[...]
@include focus-visible {
[...]
}

Reply:
This setting was made because the already existing SASS mixin (@include focus-visible) obviously shows the focus frame incompletely and in the wrong color. However, if this is actually desired in the previous way, the ":focus" directive can of course be removed again.

Reply:
If `focus-visible` has the wrong color we need to change that. Would it be acceptable from an ally perspective to switch to `focus-visible`?

Reply:
As long as it is guaranteed that the focus frame is rendered adequately, completely and in the valid color, there is really nothing wrong with doing this from an a11y perspective.